cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,56 @@
+ #!/usr/bin/env bash
+
+set -xe
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function podpreset::enable_admission_plugin() {
+    local -r cluster_name=$1
+
+    docker exec "$cluster_name-control-plane" bash -c 'sed -i \
+    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
+    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
+    /etc/kubernetes/manifests/kube-apiserver.yaml'
+}
+
+function podpreset::validate_admission_plugin_is_enabled() {
+    local -r wait_time=$1
+    local -r admission_plugin=$2
+    local -r cluster_name=$3
+    local -r control_plane_container="$cluster_name-control-plane"
+
+    timeout "${wait_time}s" bash <<EOT
```

Comment:
timeout should already return non zero and fail the function in case it fails, unless 
we again caused `set -e` to be ignored

Reply:
I tested this again, it wont catch **all** errors, since we got `-e` flag up.
I tried to avoid it but I can think of another option, I will warp the `timeout until` with `if` clause,
it will catch all error w/o messing with `-e` flag.

Reply:
> I tested this again, it wont catch all errors, since we got -e flag up.
It has a contradiction i think, `set -e` means it will fail all every error, unless we mess the `set -e`, this what you mean ?

We can even see that the failure before did return 124 which is timeout event of timeout, and failed the whole script (when it ran ps -ef not on the container, and couldn't find the string)
so i am not sure i understand what you saw now, that you are saying it won't catch errors ? (unless we just recently messed up the `set -e` with the latest changes)

I will strive to keep it as simple as possible please.

Reply:
I mean that because we use `-e` flag in case `timeout` finished with an error we didn't get to the line with `[ $? -ne 0 ] ...`
`-e` flag works as expected here.
I warped `timeout until` this with `if` clause to catch those errors,
In case `timeout` fail with 124 we would get informative error message.

Reply:
Cool, I believe we can simplify the function, but not needed in this PR.